tests/TestCli.cpp

```diff
@@ -20,7 +20,10 @@
 #include "config-keepassx-tests.h"
 #include "core/Bootstrap.h"
 #include "core/Config.h"
+#include "core/Entry.h"
 #include "core/Global.h"
+#include "core/Group.h"
+#include "core/Metadata.h"
```

Comment:
Any reason for these additions?

Reply:
@droidmonkey same reason as above I think. Importing `MainWindow.h` from `Bootstrap` allowed us not to import those explicitly.